book/src/kubernetes-changelog.md

```diff
@@ -14,6 +14,7 @@ details on individual features, visit the [Features section](features.md).
 
 ### Features
 * GA
+    * Support for [CSI spec 1.2.0](https://github.com/container-storage-interface/spec/releases/tag/v1.2.0).
```

Comment:
We may want to clarify exactly which parts of the new spec we support, since k8s currently doesn't implement everything. Cc @gnufied for suggestions